tests/test-create-delete-idempotent-%image.output

```diff
@@ -0,0 +1,2 @@
+success
```

Comment:
These reference output files are named `.golden.output`, the e2e tests won't look at just `.output` files.